src/mesido/workflows/io/write_output.py

```diff
@@ -465,7 +540,7 @@ def _name_to_asset(name):
 
         kpis_top_level.kpi.append(
             esdl.DistributionKPI(
-                name="OPEX breakdown [EUR]",
+                name="OPEX breakdown [EUR] (year 1)",
```

Comment:
I would replace "year 1" with "yearly" otherwise it seems that this is only for the first year.

Reply:
Agreed. Changed to "yearly averaged"